File: python/leetle/20250101/fizzbuzz.py
def solve(n):
  result = []
  for i in range(1,n+1):
    if i % 15 == 0:
      result.append("FizzBuzz")
    elif i % 5 == 0:
      result.append("Buzz")
    elif i % 3 == 0:
      result.append("Fizz")
    else:
      result.append(i)

  return result

print( solve(15) )

File: java/fizzbuzz/FizzBuzz.java
public class FizzBuzz {
  public static void main( String [] args ) {
    for( int i = 1; i <= 100; i++ )
    {
      System.out.println( (i%15) == 0 ? "FizzBuzz" : (i%3 == 0) ? "Fizz" : (i%5) == 0 ? "Buzz" : i );
    }
  }
}
